packages/run-protocol/src/vaultFactory/vault.js

```diff
@@ -258,7 +290,7 @@ export const makeInnerVault = (
     return makeRatioFromAmounts(collateralValueInRun, getDebtAmount());
   };
 
-  const snapshotState = (vstate, collateralizationRatio) => {
+  const snapshotState = (phase, collateralizationRatio) => {
```

Comment:
not sure I like "phase" when it's really the state for a state machine

Reply:
The motivation is for when we're on virtual objects. Writing `state.state = VaultState.ACTIVE` was worse :)